app/src/main/java/com/rpeters/jellyfin/ui/player/VideoPlayerScreen.kt

```
@@ -1018,12 +1026,12 @@ private fun VideoControlsOverlay(
                             Icon(
                                 imageVector = Icons.Default.Speed,
                                 contentDescription = "Playback speed",
-                                tint = Color.White,
+                                tint = playerColors.chipContent,
                             )
                             Spacer(modifier = Modifier.size(6.dp))
                             Text(
                                 text = String.format("%.2fx", playerState.playbackSpeed),
```

detekt (1.23.8)

[warning] String.format("%.2fx", playerState.playbackSpeed) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)